### PR TITLE
Issue 6,7 & 10 :  Update User Guide, AboutUs and PPP

### PR DESCRIPTION
**Pull request by goel-a**
#6
#7 
#10 
Update user guide to reflect new product.
Update AboutUs and PPP with goel-a's information

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,16 +9,6 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-### John Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Project Advisor
-
```
Comment (zzhikai) [inline docs/AboutUs.md:21]: Can add this back first as placeholder, in case we adding our Tutor later

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,192 +1,147 @@
----
```
Comment (zzhikai) [inline docs/UserGuide.md:1]: I think we should consider keeping most of the previous structure first for v1.1, before implementing our own formatting?

**Review by zzhikai — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (goel-a) [reply docs/AboutUs.md]: Re-added

**Review by zzhikai — APPROVED** [review]
LGTM, good work!

For future reference, when our commands has more variations. We can consider using the bulleted list for "Examples" similar to in the referenced user guide.
Screenshot 2022-02-27 at 9 55 19 PM

**Pull request merged**